W-12520153-new-pricing (#161)
mulesoft/docs-general: modules/ROOT/pages/usage-reports.adoc

A Mule flow is a sequence of logical operations configured within the XML <flow/> element of a Mule application. The report tracks the Mule flows within a deployed and running Mule application that contain a Mule event source or route APIKit requests. The report does not track other Mule flows. You can view the maximum number of Mule flows in a given month in the usage report. To calculate usage, MuleSoft tracks the number of Mule flows for all business groups, environments, and applications. The maximum number of Mule flows within a month is identified based on the peak hour across the month. For the detailed breakdown, MuleSoft shows the peak hour usage per business group, environment, and application. 

Data throughput is all the network I/O bytes produced by the infrastructure that starts and runs the Mule runtime engine that runs a Mule application. You can view the total amount of GB usage in a given month by business group, environment, and application. To calculate usage, MuleSoft tracks GB usage and aggregates the number of tracked Mule messages on a monthly basis.

> 

Author: ```suggestion

[NOTE]
--
The report lists the IDs of applications and environments. 
--

```

Runtime reports track your use of Mule messages, Mule flows, and data throughput. 

> To track Mule message usage, the report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given month by business group, environment, and application. To calculate usage, MuleSoft meters and aggregates the number of messages on a monthly basis. After a message is triggered, the report does not track changes to the message as it is processed within the application’s flows.

Reviewer: ```suggestion
To track Mule message usage, the runtime report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given month by business group, environment, and application. To calculate usage, MuleSoft meters and aggregates the number of messages on a monthly basis. After a message is triggered, the report does not track changes to the message because the message is processed within the application’s flows.
```

> A Mule flow is a sequence of logical operations configured within the XML <flow/> element of a Mule application. The report tracks the Mule flows within a deployed and running Mule application that contain a Mule event source or route APIKit requests. The report does not track other Mule flows. You can view the maximum number of Mule flows in a given month in the usage report. To calculate usage, MuleSoft tracks the number of Mule flows for all business groups, environments, and applications. The maximum number of Mule flows within a month is identified based on the peak hour across the month. For the detailed breakdown, MuleSoft shows the peak hour usage per business group, environment, and application. 

Reviewer: ```suggestion
A Mule flow is a sequence of logical operations configured within the XML `<flow/>` element of a Mule application. The runtime report tracks the Mule flows within a deployed and running Mule application that contains a Mule event source or APIKit route requests. This report does not track other Mule flows. You can view the maximum number of Mule flows in a given month in the usage report. To calculate usage, MuleSoft tracks the number of Mule flows for all business groups, environments, and applications. The maximum number of Mule flows within a month is identified based on the peak hour across the month. For the detailed breakdown, MuleSoft shows the peak hour usage per business group, environment, and application. 
```

The report lists the IDs of applications and environments. 
--

==== Supported Mule Runtime Versions

To view usage reports for Mule Runtime, all of your relevant applications (in production and sandbox) must use the following Mule versions:

[cols="2*"]
|===
|Deployment Type |Version Requirements

|CloudHub 2.0
|The latest date-patch versions for 4.3 and 4.4

|CloudHub 1.0
|The latest date-patch versions for 3.9, 4.2, 4.3 and 4.4

|Runtime Fabric
a| One of the following versions or any later date-patch:

Mule 4 images version:

* 4.4.0-20230111-1
* 4.3.0-20221212-2
* 4.2.2-20221027-7
* 4.2.1-20221027-7
* 4.2.0-20221027-6

Mule 3 images version:

* 3.9.5-20230424-2
* 3.9.4-20230424-1
* 3.9.3-20230424-1
* 3.9.2-20230424-1
* 3.9.1-20230424-1

|=== 

> You must redeploy your apps with a supported version, either manually or during the monthly patching cycle. The monthly patching cycle is applicable only for CloudHub 1.0 and CloudHub 2.0 applications. Runtime Fabric customers must be using Runtime Fabric agent version 1.13.112 or 2.1.30 or later. If usage reports do not appear properly, you must also manually redeploy Runtime Fabric applications (with a supported date-patch version).

Reviewer: Can you remove "also" from "you must also"?

> If you want to xref:cloudhub-2::ch2-update-apps.adoc#roll-back-to-a-previous-successful-configuration[roll back] to a previous successful configuration of existing applications that use a Mule version that usage reports do not support, you must choose a compliant version before deploying changes. After deployment, a new configuration with the previously chosen settings and a supported runtime is created.

Reviewer: ```suggestion
To xref:cloudhub-2::ch2-update-apps.adoc#roll-back-to-a-previous-successful-configuration[roll back] to a previous successful configuration of existing applications that use a Mule version that usage reports do not support, choose a compliant version before deploying changes. After deployment, MuleSoft creates a new configuration with the previously chosen settings and a supported runtime.
```

== Enable Access to Usage Reports

> To grant user access to usage reports, an organization administrator must grant the View Usage Reports permission.

Reviewer: I think that View Usage Reports should be in backticks.

Author: We do not usually stylize permissions.

> To grant user access to usage reports, an organization administrator must grant the View Usage Reports permission.

Author: ```suggestion
To grant user access to usage reports, an organization administrator must grant the Usage Viewer permission.
```
